config/crd/bases/_.yaml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Do not commit this file... this is some spurious file that is getting generated, not sure why (in my local setup as well)